docs/src/index.md

```diff
@@ -0,0 +1,75 @@
+# Introduction to Revise
+
+`Revise.jl` may help you keep your Julia sessions running longer, reducing the
+need to restart when you make changes to code.
+With Revise, you can be in the middle of a session and then update packages, switch git branches
+or stash/unstash code,
+and/or edit the source code; typically, the changes will be incorporated
+into the very next command you issue from the REPL.
+This can save you the overhead of restarting, loading packages, and waiting for code to JIT-compile.
+
+## Installation
+
+You can obtain Revise using Julia's Pkg REPL-mode (hitting `]` as the first character of the command prompt):
+
+```julia
+(v0.7) pkg> add Revise
+```
+
+or with `using Pkg; Pkg.add("Revise")`.
+
+## Usage example
+
+```julia
+(v0.7) pkg> add Example
```

Comment:
Packages added with `add` are technically considered immutable so would it make sense to use `dev` here instead?